.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,44 @@
+---
+name: New Component
+about: Interested in contributing a new component to Fabric? This template includes necessary information to get started, and steps to completion
+
+---
+<!-- Use this template for new components or new component varients -->
+
+## Component Details
+
+#### Exports
```

Comment:
We probably also want a "Granularity" section which asks

Is this component broken down into granular reusable units? Consider avoiding building large, rigid components, and instead building composites based on more granular generalized ones.